R/isfiletext.R

```diff
@@ -0,0 +1,41 @@
+#' Determine whether a file is "plain-text" or some sort of binary format
```

Comment:
Sorry, missed this on previous review. I like explicit `@title` and `@description` tags here.

Reply:
Done